api/tests/test_categories.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from api.tests.factories import CategoryFactory
+
+
+@pytest.mark.django_db
+def test_categories_can_be_listed(client):
+    num_items_to_list = 3
+    CategoryFactory.create_batch(num_items_to_list)
+
+    response = client.get('/api/categories/')
+    assert response.status_code == 200
+    assert response.data['count'] == num_items_to_list
+
+
+@pytest.mark.django_db
+def test_a_single_category_can_be_retrieved_by_id(client):
```

Comment:
clarity about which APIs are needed will be provided by some sequence diagrams involving `jats-ingester`, the front end, etc.